tests/blockchain/regtest/RenameWallet.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.ko
+
+#include "ottest/fixtures/blockchain/Regtest.hpp"  // IWYU pragma: associated
+
+#include <gtest/gtest.h>
+#include <thread>
+
+#include "internal/identity/Nym.hpp"
+#include "ottest/fixtures/blockchain/RegtestSimple.hpp"
+#include "ottest/fixtures/paymentcode/VectorsV3.hpp"
+#include "ottest/fixtures/ui/AccountTree.hpp"
+
+namespace ottest
+{
+const std::string name_andrew = "Andrew";
+const std::string name_bob = "Bob";
+
+TEST_F(Regtest_fixture_simple, start_chains_and_connect)
```

Comment:
Why we still want this type of test. Probably it can be in one test. Unfortunately no we can have one test per file (because of some global initialization in main test function) but I think better to make it in one test.